domino/airflow/_operator.py

```diff
@@ -89,6 +90,8 @@ def execute(self, context: Any) -> dict:
                     "multipart command string this long if is_direct=True"
                 )
 
+        
+        self.client._logger.setLevel(logging.ERROR)
```

Comment:
It would be great if you can add a property in `Domino` class
```
    @property
    def logger(self):
        return self._logger
```
and use it here like
```
self.client.logger.setLevel(logging.ERROR)
```